src/main/java/org/opentripplanner/middleware/controllers/api/MonitoredTripController.java

```diff
@@ -43,6 +48,12 @@ public MonitoredTripController(String apiPrefix) {
     protected void buildEndpoint(ApiEndpoint baseEndpoint) {
         // Add the api key route BEFORE the regular CRUD methods
         ApiEndpoint modifiedEndpoint = baseEndpoint
+            .get(path("/acceptdependent")
```

Comment:
Is the old design the reason why this endpoint is under the monitored trip controller? I think it should be under the user controller, now that dependents/guardians are linked at a user account level. Nonetheless, we should probably detect and let users know when they are assigned to a trip.

Reply:
Updated in: https://github.com/ibi-group/otp-middleware/pull/256/commits/650517476a905057e7dacafdfe034d547baec868